task/rpmbuild.yaml

```diff
@@ -129,11 +129,16 @@ spec:
             -v "$HOMEDIR/results:/results"
             --privileged --rm -ti "$mock_img"
         )
+        
+        remote_cmd chmod o+rw "$HOMEDIR/source"
+        remote_cmd chmod o+rw "$HOMEDIR/results"
```

Comment:
Making it world-writable is not optimal.  Can you please try `podman unshare` and `setfacl` to only set permissions for `mockbuilder:mock` (I think that would be 1000:135)?